src/main/java/com/easypost/model/Rate.java

```diff
@@ -20,6 +20,7 @@ public class Rate extends EasyPostResource {
     private Number estDeliveryDays;
     private String shipmentId;
     private String carrierAccountId;
+    private String billingType;
```

Comment:
Does the smartrate object need this too?

Reply:
Doesn't look like it. There's no `billing_type` key in the JSON response for the "smartrate" endpoint (last entry in this cassette: https://github.com/EasyPost/easypost-java/blob/master/cassettes/shipment/lowest_smartrate.json)

Reply:
I also verify in the cURL command, no `billing_type` was return in the JSON response

Reply:
This needs to be removed then.